README.md

```diff
@@ -213,3 +213,17 @@ The score should be an integer 0-100.
   - If invalid data supplied
     - `{"success": false, "message": "Aptitude score not added - invalid data provided.", "data": []}`
   
+
+**/addCourse**
+
+POST
+- Adds a new course to the 'courses' table in the database.
```

Comment:
To be consistent with the rest of the readme I think the quotes around courses should be removed